### DISCUSSION TITLE
Does the use of VUEX in NUXT.JS reduce the performance of the program?

### DISCUSSION THREAD

**Discussion by bianliuzhu** [nuxt/nuxt#10589] [General]
During our code review, someone that using vuex will reduce the performance of the program. I'm not sure about this issue. I want to hear everyone's opinions.

**Comment by TheAlexLichter** [comment]
TL;DR - ![It depends](https://media.giphy.com/media/xiSZGYjpYgzkipNS7x/giphy.gif)

---

Using Vuex per se does not lead to a performance decrease. It will add a little to the bundle size (because you have to load the library) but besides that, it all comes down to the actual implementation.
